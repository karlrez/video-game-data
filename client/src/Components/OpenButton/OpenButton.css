.open-button__icon {
  height: 16px;
  width: auto;
  border-radius: 50%;
}

.open-button__icon:hover {
  transform: scale(1.2);
}
