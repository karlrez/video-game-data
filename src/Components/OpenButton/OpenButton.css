.open-button__icon {
  height: 16px;
  width: auto;
  border-radius: 50%;
}
