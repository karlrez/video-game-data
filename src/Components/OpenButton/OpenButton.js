import React from "react";
import PlusIcon from "../../Assets/plus.png";
import MinusIcon from "../../Assets/minus.png";
import "./OpenButton.css";

// Button component that changes between plus and minus after clicking

function OpenButton(props) {
  return (
    <button onClick={props.onClick} className="open-button">
      {props.plusIcon ? (
        <img src={PlusIcon} alt="Plus icon" className="open-button__icon" />
      ) : (
        <img src={MinusIcon} alt="Minus icon" className="open-button__icon" />
      )}
    </button>
  );
}

export default OpenButton;
