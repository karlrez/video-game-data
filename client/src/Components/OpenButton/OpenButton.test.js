import { configure, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";
import OpenButton from "./OpenButton";

configure({ adapter: new Adapter() });

let wrapper;
beforeEach(() => {
  wrapper = shallow(<OpenButton />);
});

afterEach(() => {
  wrapper.unmount();
});

describe("Game component", () => {
  it("Matches snapshot", () => {
    expect(wrapper).toMatchSnapshot();
  });
});
