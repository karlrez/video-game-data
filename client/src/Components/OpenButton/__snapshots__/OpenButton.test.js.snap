// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Game component Matches snapshot 1`] = `ShallowWrapper {}`;
